RPIPs/RPIP-44.md

```diff
@@ -52,5 +52,14 @@ The exit functionality for a megapool is specified as follows:
 - The protocol SHALL use the `withdrawn`, `credit`, and `received` balances to pay off `debt` prior to taking action on exits
 - The protocol SHOULD use staked RPL to decrease `debt` by the corresponding amount prior to taking action on exits
 
+## Security Considerations
+- A misbehaving oDAO gains the ability to force exit any validator by applying penalties to increase `deficit` beyond `exit_deficit`. Alongside a malicious protocol upgrade, this would allow for control of all principal.
+  - The penalty application can be limited, eg, by using a maximum rate as suggested in [RPIP-58](RPIP-58.md)
+  - The malicious upgrade risk can bre reduces, eg, via security council veto as proposed in [RPIP-60](RPIP-60.md)
+- If a bad actor gains control of a node operator key, they are now able to exit validators
+  - This doesn't result in the loss of funds, so is not too problematic
```

Comment:
this is true if withdrawal address is different from node address, not generally though